pages/common/docker-login.md

```diff
@@ -0,0 +1,12 @@
+# docker login
+
+> Log into a docker registry.
+> More information: <https://docs.docker.com/engine/reference/commandline/login/>.
+
+- Log into registry with username and password:
+
+`docker login -u {{username}} -p {{password}} {{server}}`
+
+- Log into registry with password from stdin:
+
+`{{cat path/to/password.txt}} | docker login -u {{username}} --password-stdin`
```

Comment:
```suggestion
` echo "{{password}}" | docker login -u {{username}} --password-stdin`
```

Maybe echo instead

Reply:
If you are echoing the string, you might as well just use the -p argument.

On the other hand it's just to show how it accepts pipes to stdin so we could use any command.

Reply:
Just a suggestion, however I would prefer echo since (if this tool works on windows) then echo is more universal

Reply:
Windows..... :vomiting_face: 

Powershell doesn't support echo, and the cat command is a unix standard, ever since version 1, and standardized by IEEE.

Reply:
If you just look at my README personal, you will find that my favourite one is a great from Mr Torvalds

Reply:
XD That's a fantastic quote. I approve.

Reply:
>Just a suggestion, however I would prefer echo since (if this tool works on windows) because echo is more universal

Yep, `echo` is available in cmd and powershell, so let's use it.

>If you just look at my README personal, you will find that my favourite one is a great from Mr Torvalds

Windows is what enterprises use, so there's no running away from it for everybody for a long time—and it has it's fun things too, to be honest—but that quote is really, really funny @CleanMachine1 😆